### DISCUSSION TITLE
Icon theme not working in i3

### DISCUSSION THREAD

**Discussion by JavaIsNotHard** [i3/i3#5869] [Q&A]
Icon themes are not working in i3 even thought I have set them through lxappearance.
![Fri Jan 19 08:16:01 PM +0545 2024](https://github.com/i3/i3/assets/96734085/4775780b-0ada-4556-8e35-b717c78f2a12)

**Comment by orestisfl** [comment]
They do work but not much to be done about your issue as there is no information. Try logging out and in again as this might work.

**Reply by JavaIsNotHard** [reply]
Yeah I have tried both logging out and rebooting but it still does not work. What do you think could be the issue?

**Comment by mikeredev** [comment]
set it in `gnome-tweaks`

then verify/add in your `~/.config/gtk-3.0/settings.ini` this line `gtk-icon-theme-name=[Icons]`

if not working after reboot, what file manager are you using?

**Reply by JavaIsNotHard** [reply]
I am using nautilus currently and it seems that the icon theme is not being applied to gtk-4.0 applications but it works for gtk3.0 applications. Also gtk4.0 theme is not being applied.
